src/h5web/toolbar/Toolbar.module.css

```diff
@@ -56,6 +56,11 @@
   font-size: 0.875em;
 }
 
+.btn[data-raised] > .btnLike {
+  background-color: var(--white);
+  box-shadow: 0 0 1px var(--btn-shadow-color);
+}
```

Comment:
I needed a toggle with a background when idled (which the toggles in the toolbar don't have).